### ISSUE TITLE
Name property for Tree item in Left pane for 'View Inspect' tab items is defined incorrectly 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#352]
### Steps to Reproduce

1. Open "Xamarin Workbook"
2. Products window is opened.
3. Navigate to the "wpf "icon and select "create workbook".
4. Navigate to tree items present in left side pane under 'View Inspect' tab item and  Observe Inspect property

### Expected Behavior
Name property should be properly defined for Tree item(For e.g -'Hello Workbooks' )
### Actual Behavior
Name property for Combo box is defined as 'Xamarin.Interactive.Client.ViewInspector.InspectTreeNode'

Narrator Behavior: Narrator is announcing as 'Xamarin.Interactive.Client.ViewInspector.InspectTreeNode, 3 of 3, selected'
NVDA behavior:NVDA is announcing as 'Tree view
Xamarin.Interactive.Client.ViewInspector.InspectTreeNode, level 5'

### User Impact:
User who rely on screenreader will not get confused while traversing through tree item in left side pane

### Screen-Shots: 
![tree_item_view_inspect_tab_inspect](https://user-images.githubusercontent.com/22956130/40723687-0ba7e130-643d-11e8-8417-0233d8086d53.png)
![tree_item_view_inspect_tab_narrator](https://user-images.githubusercontent.com/22956130/40723688-0bd99ac2-643d-11e8-92c7-67482e0c1a8b.png)
![tree_item_view_inspect_tab_nvda](https://user-images.githubusercontent.com/22956130/40723691-0c0845ca-643d-11e8-8232-6797e8c6f6d1.png)

### Environment: 
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
